PrestaShop/PrestaShop
# Refresh documents tab when adding product in order view page

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | When adding a product and choosing to create a new invoice, the "Documents" tab wasn't updated with the new invoice. This PR fixes it.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #17977
| How to test?  | See below

## How to test

Please rebuild assets before testing.

1. Go to BO > orders > orders > View order<br>2. Click on add a product<br>3. Select a product and select "Create a new invoice"<br>4. Click on add<br>5. The new invoice should be displayed in the documents tab, the number of documents should be updated as well.




---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/18196)
